### PR TITLE
Release aug

### PR DESCRIPTION
**Pull request by patryllus**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/hiv/hivEnrollment.html [hunk 1]
```diff
@@ -172,11 +172,18 @@
                 getField('encounter-date.error').html('Enrollment date should not be empty').show();
                 return false;
             }
+
+            if ((transferStatus == NO_CONCEPT_ID || transferStatus == YES_CONCEPT_ID) &amp;&amp; confirmedPveDate =="") {
+                getField('confirmed-positive-date.error').html('Confirmed HIV+ date, should not be empty').show();
+                return false;
+            }
+
```
Comment (ojwanganto) [inline omod/src/main/webapp/resources/htmlforms/hiv/hivEnrollment.html]: @patryllus can you explain this change?

File: omod/src/main/webapp/resources/htmlforms/hiv/hivEnrollment.html [hunk 2]
```diff
@@ -211,9 +218,14 @@
             }
 
             var phoneno = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/;
-            if((tsTelephone != "" || tsTelephone != null) &amp;&amp; !tsTelephone.match(phoneno)) {
-                getField('ts-telephone.error').html('Phone number should contain only digits').show();
-                return false;
+
+            if(transferStatus == NO_CONCEPT_ID || transferStatus == YES_CONCEPT_ID) {
```
Comment (ojwanganto) [inline omod/src/main/webapp/resources/htmlforms/hiv/hivEnrollment.html]: What exactly is to be achieved by this line?

File: api/src/main/resources/content/kenyaemr.common.xml [hunk 3]
```diff
@@ -153,7 +153,7 @@
 				<ref bean="kenyaemr.app.clinician" />
 				<ref bean="kenyaemr.app.chart" />
 				<ref bean="kenyaemr.app.hts" />
-				<ref bean="kenyaemr.app.prep" />
+				<!--<ref bean="kenyaemr.app.prep" />-->
```
Comment (ojwanganto) [inline api/src/main/resources/content/kenyaemr.common.xml]: The commit message doesn't talk to the change on this file

**Comment by patryllus** [comment]
OVERTAKEN BY EVENTS

**Pull request closed without merging**